lib/teiserver/account/libs/relationship_lib.ex

```diff
@@ -298,23 +299,12 @@ defmodule Teiserver.Account.RelationshipLib do
 
   @spec check_block_status(T.userid(), [T.userid()]) :: :ok | :blocking | :blocked
   def check_block_status(userid, userid_list) do
-    user = Account.get_user_by_id(userid)
     userid_count = Enum.count(userid_list) |> max(1)
 
-    {block_count_needed, block_percentage_needed} =
-      cond do
-        user.behaviour_score <= 5000 ->
-          {2, 20}
-
-        user.behaviour_score <= 8000 ->
-          {3, 30}
-
-        true ->
-          {
-            Config.get_site_config_cache("lobby.Block count to prevent join"),
-            Config.get_site_config_cache("lobby.Block percentage to prevent join")
-          }
-      end
```

Comment:
This doesn't seem relevant to for the avoid balancer. Should be done as a different PR, but IMO it's already fine as is.